lib/slack_message/api.rb

```diff
@@ -41,6 +41,10 @@ def post(payload, target, profile, time)
       raise ArgumentError, "Tried to send an entirely empty message."
     end
 
+    if params[:blocks].length > 50
+      raise ArgumentError, "Message cannot contain more than 50 blocks"
```

Comment:
I should be more consistent on these, but anyway let's not make it worse.

```suggestion
      raise ArgumentError, "Message cannot contain more than 50 blocks."
```